internal/backend/svc/svc.go

```diff
@@ -320,7 +320,9 @@ func NewOpts(cfg *Config, ropts RunOptions) []fx.Option {
 	return append(opts, fx.Populate(svcs))
 }
 
-func StartDB(ctx context.Context, cfg *Config) (db.DB, error) {
+func StartDB(ctx context.Context, cfg *Config, ropt RunOptions) (db.DB, error) {
+	setFXRunOpts(ropt)
```

Comment:
how mode is influence db? it's always persistent, no? I thought that the only things are relevant are DSN and type

Reply:
Mode dev hides the default db location 
So I used the same logic, instead of duplicating it
That's the same usage when starting the server so I wanted to have the same here experience